sktm/jenkins.py

```diff
@@ -401,23 +438,16 @@ def _params_eq(self, build, params):
 
     def find_build(self, params, eid=None):
         job = self.__get_job()
-        lbuild = None
-
-        while not lbuild:
-            try:
-                lbuild = job.get_last_build()
-            except jenkinsapi.custom_exceptions.NoBuildData:
-                time.sleep(1)
+        lbuild = self.__get_last_build(job, 1)
```

Comment:
I'm not sure if this is an equivalent change. The purpose of the removed code was to poll the Jenkins server until the build appears. The retries should solve things like temporary outages, which this is not. Retrieval of the build data might take more time than the retries specified, and that should not be a problem, but after this change, sktm would quit.

Reply:
Well, the related code is rolled back.